io/net/iperf_test.py

```diff
@@ -200,10 +200,12 @@ def test(self):
                 tput = int(line.split()[6])
             elif id in line and 'Gbits/sec' in line:
                 tput = int(float(line.split()[6])) * 1000
+            elif id in line and 'Kbits/sec' in line:
+                tput = int(float(line.split()[6])) / 1000
         if tput < (int(self.expected_tp) * speed) / 100:
             self.fail("FAIL: Throughput Actual - %s%%, Expected - %s%%"
                       ", Throughput Actual value - %s "
-                      % ((tput*100)/speed, self.expected_tp,
+                      % (round((tput*100)/speed, 4), self.expected_tp,
```

Comment:
@vaishnavibhat The result of all the speed types like Kbits/sec or Mbits/sec or Gbits/sec are converted to Mbits/sec using a formula before printing and then a common results in Mbits/sec is printed.

In case to print all the speed types as suggested, need to change the formula and calculate throughput for each different value and then have to print the results would make it more complicated i feel. And expected throughput value is given in yaml file as input, which is as per the formula given in Mbits/sec only.

Plus the results in Mbits/sec give an idea how less the throughput from expected value. 

@vaishnavibhat  @abdhaleegit  Please let me know should we go for these changes.

Reply:
Okay Tasmiya, 
I see the results are printed in the below format on the console and is user friendly. 
(1/9) iperf_test.py:Iperf.test;run-hbond-mtu-1500-bdec: FAIL: FAIL: Throughput Actual - 9.44%, Expected - 90%, Throughput Actual value - 944Mb/sec (41.07 s)

I am okay to hold on to any more changes for now.

Reply:
LGTM